test/kubemark/resources/hollow-node_template.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: hollow-node
   labels:
     name: hollow-node
+    {{KUBEMARK_MIG_CONFIG}}
```

Comment:
I think we're using lowercase for template variables and capitals for env vars in other files.

Reply:
Done.